website/content/sdk/guides/intros/stocks/options.md

```diff
@@ -1,82 +1,110 @@
 ---
 title: Options
-keywords: ["stocks", "options", "calls", "puts", "gamma", "delta", "iv", "theta", "rho", "greeks", "charm", "vanna", "vomma", "derivatives", "contracts", "^SPX", "^VIX", "^NDX", "chains", "oi", "vol", "volume", "open", "interest", "expiration", "dte", "volatility", "underlying"]
-excerpt: "This guide introduces the Stock Options module, within the context of the OpenBB SDK"
-geekdocCollapseSection: true
 ---
-The Stock Options module of the OpenBB SDK wraps the functions of the Options sub-menu within the [OpenBB Terminal](/terminal/guides/intros/stocks/options). This allows the user greater flexibility when researching and anlayzing options data. Start a Python script of Notebook file by importing the dependencies:
 
-```python
-from openbb_terminal.sdk import openbb
-
-import pandas as pd
-import numpy as np
-
-import matplotlib.pyplot as plt
-
-%matplotlib widget
-```
+Please be advised that this module is subject to improvements, the guide here may not reflect its current status.
```

Comment:
Please remove this, we don't have to set expectations for the audience. There is **always** room for improvement, nothing is perfect.